feat: add vuetify

Amaaq opened the pull request:
fixes #1

Commit:
feat: add vuetify

maaqoul commented on package.json:
```diff
@@ -10,7 +10,10 @@
   },
   "dependencies": {
-    "vue": "^3.3.4"
+    "@chakra-ui/vue": "^0.13.1",
```
why didn't you remove the chakra ui and emotion/css dependency?

Amaaq replied:
actually i didn't remove it because i tought it is something that i would do later on

maaqoul commented on src/App.vue:
```diff
@@ -1,4 +1,5 @@
-<script setup>
```
why this has been changed to lang=js?

Amaaq replied:
i'm not sure about this honestly i don't recall where that came from

Amaaq committed:
devDependencies and script tag corrected

chakraUi and emotion/css devDependencies uninstalled and lang attribute changed

fixes #1
